spaceship/lib/spaceship/connect_api/client.rb

```diff
@@ -347,8 +348,25 @@ def add_beta_groups_to_build(build_id: nil, beta_group_ids: [])
         handle_response(response)
       end
 
+      def request(method, url_or_path = nil, params = nil, headers = {}, auto_paginate = false, &block)
+        headers['Authorization'] = "Bearer #{token.text}"
```

Comment:
We probably need to add an `if token` around this line because there won't always be a `token` if we are using the web cookie sessions like _fastlane_ uses now 🤔